### PR TITLE
Further steps into synthetic homotopy theory

### PR DESCRIPTION
**Pull request by plt-amy**
I was pushing towards E-M spaces but it's a bit of a pain so I'm going to cut this PR here and pick up the work afterwards. (Also writing a PR so I can be better aware of what still needs to be written).

Featuring:
- A new home for deloopings
- Cutting up Homotopy.Base
- Relative characterization of n-connectedness + the "elimination principles" relative to n-types
- Conjugations of paths
- "Wedge connectivity"

### CODE REVIEW

File: src/Homotopy/Conjugation.lagda.md
[the remark this answers is not in the document]
Comment (ncfavier) [reply src/Homotopy/Conjugation.lagda.md:1]: (I can take care of harmonising our definitions later if you just want to proceed with this PR; I'd like to see it land!)

**Comment by plt-amy** [comment]
i guess we can merge main into this then merge it as is?

**Comment by ncfavier** [comment]
Sure

File: src/HoTT.lagda.md [hunk 1]
```diff
@@ -626,6 +637,26 @@ _ = ΩS¹≃integers
...
+
+### 8.6 The Freudenthal suspension theorem
+
+<!--
+```agda
+_ = relative-n-type-const-plus
+```
+-->
+
+* Lemma 8.6.1: `relative-n-type-const-plus`{.Agda}
```
Comment (ncfavier) [inline src/HoTT.lagda.md:658]: We can add wedge connectivity (lemma 8.6.2)

File: src/Homotopy/Wedge.lagda.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+open import Homotopy.Connectedness
+```
+-->
+
+```agda
+module Homotopy.Wedge where
+```
+
+# Wedge connectivity
```
Comment (ncfavier) [inline src/Homotopy/Wedge.lagda.md:15]:
```suggestion
# Wedge connectivity

The **wedge connectivity lemma** is used in the proof of the Freudenthal suspension theorem.
```

Better than nothing?

**Review by ncfavier — APPROVED** [review]

**Pull request merged**